src/static/css/pages/careers.less

```diff
@@ -13,7 +13,19 @@
 
 .careers-working-at-cfpb {
     .content_hero {
-        min-height: 300px;
+        background-color: @white;
+        /* TODO: This breakpoint is set to the small image width,
+                 so that a gap does not open below the image. */
+        .respond-to-min(660px, {
+            min-height: 300px;
+        });
+
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/1c8dc52bfb21abbbb293dbbf4ded5b77f38c9b68